docs/wiki/Whats-new.md

```diff
@@ -3,28 +3,115 @@
 - [In this Section](#in-this-section)
 - [Updates](#updates)
   - [May 2023](#may-2023)
+    - [Policy](#policy)
```

Comment:
```suggestion
    - [Policy](#policy)
```
It looks like you're using a VS Code markdown extension that is configured to automatically generate table of contents on save. Only the month and year should be in the TOC, and not all the subsections. Please review your extensions and revise the TOC.

Reply:
I have compared my code with the main branch and even your suggestion shows as invalid as there is no change in the code?